examples/expressions/metadata/imp.rs

```diff
@@ -0,0 +1,81 @@
+use glib::prelude::*;
+use glib::subclass::prelude::*;
+use gtk::glib;
+
+use std::cell::RefCell;
+
+#[derive(Debug)]
+pub struct Metadata {
+    pub title: RefCell<String>,
+    pub last_modified: RefCell<glib::DateTime>,
+}
+
+#[glib::object_subclass]
+impl ObjectSubclass for Metadata {
+    const NAME: &'static str = "Metadata";
+    type Type = super::Metadata;
+    type ParentType = glib::Object;
```

Comment:
I'm somehow not sure if you need to specify the ParentType in this case

Reply:
You always have to specify it

Reply:
Hmm, looks like a possible simplification of glib::object_subclass :)

Reply:
Opened an issue upstream at https://github.com/gtk-rs/gtk-rs-core/issues/350